### PR TITLE
Bench tuning

### PR DESCRIPTION
**Pull request by daron666**

### CODE REVIEW

**Comment by catena2w** [comment]
Why do we need to change `src/test/resources/bench/modifiers.txt? It's 21 Mb file that makes our git very heavy..

File: src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala [hunk 1]
```diff
@@ -190,12 +190,13 @@ trait HeadersProcessor extends ScorexLogging {
       Failure(new Error("Parent header is marked as semantically invalid"))
     } else {
       Success()
-    }.recoverWith { case thr =>
```
Comment (catena2w) [inline src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala]: What is the purpose of this change?

Comment (daron666) [reply src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala]: I removed it, cause this change is out of scope of current task. I think that this `recoverWith` will be only applied to `Success` and hence never will be called.

Here is link to small example that demonstrates that `recover` in `validate` won't work, rather than in `validateF` [Scastie](https://scastie.scala-lang.org/uFFsJoxfQqu6isDhZinrjg)
